install/deb/templates/web/nginx/hosting.tpl

```diff
@@ -14,7 +14,12 @@ server {
             try_files      $uri @fallback;
         }
     }
-
+    
+    location ~ [^/]\.php(/|$) {
+        fastcgi_param SCRIPT_FILENAME $document_root$fastcgi_script_name;
+        fastcgi_param PHP_ADMIN_VALUE "sendmail_path = /usr/sbin/sendmail -t -i -f admin@%domain%";
```

Comment:
i was wandering about the following code:
```
if (-f /usr/sbin/sendmail){

        fastcgi_param PHP_ADMIN_VALUE "sendmail_path = /usr/sbin/sendmail -t -i -f admin@%domain%";
}
```
cause the system might not have the sendmail in that path somehow or not installed